src/xdocs/config_coding.xml

```diff
@@ -4303,6 +4320,32 @@ for(int i=0; i&lt;10; i++) {
   &lt;property name=&quot;max&quot; value=&quot;3&quot;/&gt;
 &lt;/module&gt;
         </source>
+        <p>Valid Code Example:</p>
+        <source>
+if (x &gt; 2) {
+   if (x &gt; 3) {
+      if (x &gt; 4) {
+         if (x &gt; 5) {}
+         else {} // OK
+      }
+   }
+}
+        </source>
+        <p>Invalid Code Example:</p>
+        <source>
+if (x &gt; 2) {
+   if (x &gt; 3) {
+      if (x &gt; 4) {
+         if (x &gt; 5) {
+            if (x &gt; 6) { // violation, depth exceeded '3'
+               if (x &gt; 7) {}
```

Comment:
Nested if-else depth is 5 (max allowed is 3).

Reply:
done.

Reply:
resolved